### ISSUE TITLE
Expand supported variant endings in a phrase beyond 2

### ISSUE THREAD

**Issue by newlandsvalley** [newlandsvalley/purescript-abc-melody#10]
i.e. for ABC sections with volta descriptions of the form:

```
    ..|1 ....:|2 .....:|3 .... etc
```

**Comment by newlandsvalley** [comment]
And also of the form:

```
..|1,3.....:|2,4...:| etc
```

which is now supported in the volta branch of abc-parser

**Comment by newlandsvalley** [comment]
This is now fixed by 9985e1c14e762d765ead2323d721fe88e25673bd in the __morevariants__ branch.

**Comment by newlandsvalley** [comment]
Except that this is not fixed for variants of the form:

```
  ...|1,2,3 ....:|4....:|...
```
Here, we are using the algorithm suitable for the case above.  We are not recognising the variant ending for cases 1 and 2.

**Comment by newlandsvalley** [comment]
Fixed by 61661a0af2693f949fd5e5464e857b0982f744a5
